public/index.html

```diff
@@ -5,10 +5,12 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <script src="https://cdn.nimiq-testnet.com/web-offline.js" defer></script>
+    <script src="https://cdn.nimiq-testnet.com/web-offline.js"></script>
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title>Nimiq Accounts Manager</title>
     <link href="blocking.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Muli:400,600,700" rel="prefetch">
+    <link href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY" rel="prefetch">
```

Comment:
This doesn't work.
This really only triggers the download of the specified files but they will not be applied as stylesheets.
Also `rel="prefetch"` has a low priority. Thus, a FOUC might occur.

Sorry for causing that much back and forth.
I really just meant inlining the contents of the google stylesheet into our stylesheet.
I'll create a PR towards your branch to show what I meant.

Reply:
Ah, you're right... well, it's quite clear to me now, but go ahead :)

Reply:
@danimoh I guess you're busy with faucet stuff, so I will do it tomorrow.

Reply:
I'm actually fine with just making them `stylesheets` again and forgetting the inlining for now.

Reply:
I created the PR now: https://github.com/nimiq/accounts/pull/116